README.md

```diff
@@ -15,6 +15,39 @@ for Windows' [Google Group](http://groups.google.com/group/git-for-windows),
 and [contribute bug
 fixes](https://github.com/git-for-windows/git/wiki/How-to-participate).
 
+To build Git for Windows, please either install [Git for Windows'
+SDK](https://gitforwindows.org/#download-sdk), start its `git-bash.exe`, `cd`
+to your Git worktree and run `make`, or open the Git worktree as a folder in
+Visual Studio.
+
+To verify that your build works, use one of the following methods:
+
+- If you want to test the built executables within Git for Windows' SDK,
+  prepend `<worktree>/bin-wrappers` to the `PATH`.
+- Alternatively, run `make install` in the Git worktree.
+- If you need to test this in a full installer, run `sdk build
+  git-and-installer`.
+- You can also "install" Git into an existing portable Git via `make install
+  DESTDIR=<dir>` where `<dir>` refers to the top-level directory of the
+  portable Git.
+- If you built using Visual Studio, you can use the menu item `Build>Install
+  git` (you will want to click on `Project>CMake Settings for Git` first, then
+  click on `Edit JSON` and then point `installRoot` to the `mingw64` directory
+  of an already-unpacked portable Git).
+- If you want to run the built executables in-place, but in a CMD instead of
+  inside a Bash, you can run a snippet like this in the `git-bash.exe` window
+  where Git was built, and then paste it into the CMD window (be sure to paste
+  the `EOF` line without leading spaces):
+
+  ```sh
+  clip.exe <<EOF
+  set GIT_EXEC_PATH=$(cygpath -aw .)
+  set PATH=$(cygpath -awp ".:contrib/scalar:/mingw64/bin:/usr/bin:$PATH")
+  set GIT_TEMPLATE_DIR=$(cygpath -aw templates/blt)
+  set GITPERLLIB=$(cygpath -aw perl/build/lib)
+  EOF
+  ```
+
```

Comment:
I think I would finish up with a statement on how to confirm that they
are running their recently built version of Git.  That might be just
running `where` or `which` or whatever.   And maybe `git version --build-options`.
Just some way to help them identify that they did things correctly.
(For example, a typo when updating their PATH and they might silently
fallback to using the officially installed version.)

Reply:
Yes, that is a good point!

Reply:
Fixed.